docker-compose.yml

```diff
@@ -42,15 +46,16 @@ services:
       "Settings.saml.key_path": "/srv/vets-api/src/config/certs/vetsgov-localhost.key"
       "Settings.redis.app_data.url": "redis://redis:6379"
       "Settings.redis.sidekiq.url": "redis://redis:6379"
-      "Settings.binaries.clamdscan": "clamscan" # Not running a separate process within the container for clamdscan, so we use clamscan which requires no daemon
```

Comment:
We will probably want to restore the ability for local dev to skip clamav. Then again, it's really fast, so :shrug:.